Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM quay.io/assemblyline/nodejs:0.12.10
 RUN apt-get update -q\
```

Comment:
As david said this image is solely for building frontend js apps.  We do however support more modern nodejs versions. See: https://github.com/assemblyline/builder-frontendjs/blob/master/Assemblyfile  That said we should update those versions.

In the medium term this image will be deprecated by the newer more flexible stuff we are working on... Workflows like this will just be a plugin to a generic builder image ...